namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -61,7 +65,7 @@ private[consul] object SvcAddr {
       // a good state was seen.
       @volatile var lastGood: Option[Addr] = None
       @volatile var stopped: Boolean = false
-      def loop(index0: Option[String]): Future[Unit] = {
+      def loop(index0: Option[String], backoffs: Option[Stream[Duration]], nonFatalLogLevel: Level = Level.WARNING): Future[Unit] = {
```

Comment:
Can we make this always take a `Stream[Duration]` instead of an `Option[Stream[Duration]]` and pick the default backoffs at the call sites where appropriate?  Similarly, I think we should not provide a default log level here and be more explicit about the log level at the call sites.   Since this is a local method, this should be pretty easy.